src/ert/ensemble_evaluator/config.py

```diff
@@ -15,8 +16,8 @@
 from cryptography.hazmat.primitives import hashes, serialization
 from cryptography.hazmat.primitives.asymmetric import rsa
 from cryptography.x509.oid import NameOID
-from dns import exception, resolver, reversename
 
+from ert.shared import get_machine_name as ert_shared_get_machine_name
```

Comment:
Can this be only `from ert.shared import get_machine_name as _get_machine_name`?

Reply:
Hmm, I think i prefer the explicitness of the long name. Nonetheless, we will change this again in short time when we remove the deprecated (private) API.